src/Util/Network.php

```diff
@@ -36,7 +36,13 @@ class Network
 	 */
 	public static function fetchUrl($url, $binary = false, &$redirects = 0, $timeout = 0, $accept_content = null, $cookiejar = 0)
 	{
-		$ret = self::curl(
+		$ret = fetchUrlFull($url, $binary, $redirects, $timeout, $accept_content, $cookiejar);
+
+		return($ret['body']);
```

Comment:
Code Standards: Remove parentheses from language constructs (include, return). I know you didn't write it, but I'd like to bring the whole code base to code standards.